tests/meta/conftest.py

```diff
@@ -136,47 +137,160 @@ def analysis_store(base_store, analysis_family):
 
 
+class MockVersion:
+    """Mock a version object"""
+
+    @property
+    def id(self):
+        return ""
+
+
 class MockFile:
-    def __init__(self, path):
-        self.path = path
+    """Mock a file object"""
 
-    def first(self):
-        return MockFile()
+    def __init__(self, path="", to_archive=False, tags=None):
+        self.path = path
+        self.to_archive = to_archive
+        self._tags = []
+        if tags:
+            for tag in tags:
+                self._tags.append(MockTag(tag))
 
     def full_path(self):
         return ""
 
+    def is_included(self):
+        return False
+
+    @property
+    def tags(self):
+        return self._tags
+
+
+class MockFiles:
+    """Mock file objects"""
+
+    def __init__(self, files):
+        self._files = files
+
+    def all(self):
+        return self._files
+
+    def first(self):
+        return self._files[0]
+
+
+class MockTag:
+    """Mock a file object"""
+
+    def __init__(self, name):
+        self._name = name
+
+    @property
+    def name(self):
+        return self._name
+
+
+class MockHouseKeeper(HousekeeperAPI):
+    """Mock the housekeeper API"""
+
+    # In this mock we want to override __init__ so disable here
+    def __init__(self, files: MockFiles = None):
+        self._file_added = False
+        self._file_included = False
+        self._files = MockFiles([])
+        if files:
+            self._files = files
+        self._file = MockFile()
+
+    # This is overriding a housekeeper object so ok to not include all arguments
+    def files(self, bundle, version, tags):
+        """Mock the files method to return a list of files"""
+        files = []
+
+        for file in self._files.all():
+            if any(tag.name in tags for tag in file.tags):
+                files.append(file)
+
+        return MockFiles(files)
+
+    def get_files(self, bundle, tags, version="1.0"):
+        """Mock the get_files method to return a list of files"""
+        return self._files
+
+    def add_file(self, file, version_obj, tag_name, to_archive=False):
+        """Mock the add_files method to add a MockFile to the list of files"""
+        self._file_added = True
+        self._file = MockFile(path=file)
+        return self._file
+
+    def version(self, bundle: str, date: str):
+        """Fetch version from the database."""
+        return MockVersion()
+
+    def last_version(self, bundle: str):
+        """docstring for last_version"""
+        return MockVersion()
+
+    def include_file(self, file_obj, version_obj):
+        """docstring for include_file"""
+        self._file_included = True
+
+    def add_commit(self, file_obj):
+        """Overrides sqlalchemy method"""
+        return file_obj
+
+    def get_root_dir(self):
+        """Overrides sqlalchemy method"""
+        return self._file
+
+
+class MockLims:
+    """Mock lims fixture"""
+
+    lims = None
+
+    def __init__(self):
+        self.lims = self
+
 
 class MockDeliver:
```

Comment:
```suggestion
class MockDeliver(DeliverAPI):
```
To get the rest of the API "for free"

Reply:
Nice!